account_banking_make_deposit/wizard/add_deposit_items_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Mix of tabs and spaces here, that could be cleaned up with indents of 2 or 4 spaces to be consistent.